src/components/IconButton/IconButton.st.css

```diff
@@ -0,0 +1,55 @@
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: ButtonNext;
+}
+
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultColor: color-5;
+    DisabledColor: color-3;
+}
+
+:vars {
+    /*Overrides*/
+    IconColor: --overridable;
+}
+
+.root {
+    -st-extends: ButtonNext;
+    -st-states: disabled;
+    /*css goes here*/
+    background-color: transparent;
+    text-decoration: none;
+    border: none;
+    display: inline-block;
+    padding: 0;
+    margin: 0;
+    line-height: 0;
+}
+
+.root:hover .icon {
+    opacity: applyOpacity(value(IconColor), 0.6);;
```

Comment:
The `applyOpactiy` changes the opacity of a given color.
In this case your changing the `opacity` so it can just be `0.6`.
Meaning, you can write either:
```
opacity: 0.6;
```
or:
```
stroke: applyOpacity(value(IconColor), 0.6);
```